dataRetriever/resolvers/topicResolverSender/topicResolverSender.go

```diff
@@ -54,15 +57,21 @@ func NewTopicResolverSender(arg ArgTopicResolverSender) (*topicResolverSender, e
 	if check.IfNil(arg.OutputAntiflooder) {
 		return nil, dataRetriever.ErrNilAntifloodHandler
 	}
+	if arg.NumCrossShardPeers+arg.NumIntraShardPeers < minPeersToQuery {
```

Comment:
Should be checked also separately? -> arg.NumCrossShardPeers > 0 and arg.NumIntraShardPeers > 0 to avoid combination of 0,2 or 2,0 ?

Reply:
the combination of 0,2 and 2,0 is allowed but you are right, should check both values to be >=0 as t not have 4,-2

Reply:
done